db/migrate/20160329232816_add_pushed_to_spree_shipment.rb

```diff
@@ -0,0 +1,5 @@
+class AddPushedToSpreeShipment < ActiveRecord::Migration
+  def change
+    add_column :spree_shipments, :pushed, :boolean
```

Comment:
Leaving me a note to not forget that we should mark all shipped shipments as pushed too in this migration.